CMakeLists.txt

```diff
@@ -474,6 +484,7 @@ if(ENABLE_CONAN)
     GENERATORS cmake_find_package
     KEEP_RPATHS
     NO_OUTPUT_DIRS
+    OPTIONS tbb:shared=${TBB_SHARED}
```

Comment:
Tbh I spend several days trying to understand why `osrm-extract.exe` silently finishes with `-1073741515` exit code(Google says that it is something about DLL loading), then I found that it happens on the first access of any TBB symbol. I tried to statically compile it and it worked. I believe it is possible to make it working with dynamic linkage, but I have no Windows machine and had to debug all these things directly on CI, so debugging is really limited for me.

Reply:
:+1: 
Can you add a comment to that effect in the file?

Reply:
Added.